pagerduty-incident.yaml

```diff
@@ -0,0 +1,5 @@
+name: pagerduty-incident
+repo: https://github.com/coralogix/eng-concourse-resource-pagerduty-incident
+container_image: ghcr.io/coralogix/eng-concourse-resource-pagerduty-incident
```

Comment:
When I do `docker pull`:
```
Using default tag: latest
Error response from daemon: Head "https://ghcr.io/v2/coralogix/eng-concourse-resource-pagerduty-incident/manifests/latest": unauthorized
```
Is the image private?

Reply:
@taylorsilva ah, we don't publish `latest` tags as we consider it to be an anti-pattern (prone to cause breakages). Please pull `v0.1.0`.

Reply:
Nice! Still getting unauthorized though :( I don't use ghcr myself, do they force you to authorize now?
```
docker pull ghcr.io/coralogix/eng-concourse-resource-pagerduty-incident:v0.1.0
Error response from daemon: Head "https://ghcr.io/v2/coralogix/eng-concourse-resource-pagerduty-incident/manifests/v0.1.0": unauthorized
```

Reply:
@taylorsilva I figured it out, for some reason (who knows?) GitHub allows for private packages on public repositories (huh?).
I manually set the package visibility to public, and was able to pull locally after running `docker logout ghcr.io`.

Reply:
huh, weird! And that was free?

I can pull the image now too so this is good to go! 🚀

Reply:
@taylorsilva yeah, there's a small amount of storage and transfer included in paid GitHub plans (which we have), and then public images are free. https://docs.github.com/en/billing/managing-billing-for-github-packages/about-billing-for-github-packages

which from our perspective makes GHCR seem pretty compelling, because a) Docker Hub has its annoying pull rate limits, so nobody likes them anymore (even if we understand that it was important for them to put in...) b) Quay has gone down way too many times for us to consider them reliable anymore, c) AWS now has public ECR repositories, but discoverability doesn't feel as nice as GitHub, and there are explicit quotas, even if people generally are unlikely to meet them.